src/GovUk.Education.ExploreEducationStatistics.Content.Model/UserPublicationInvite.cs

```diff
@@ -0,0 +1,35 @@
+#nullable enable
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Model
+{
+    public class UserPublicationInvite
+    {
+        [Key] 
+        [Required]
+        public Guid Id { get; set; }
+
+        [Required] public string Email { get; set; } = null!;
+
+        [Required] public Publication Publication { get; set; } = null!;
+
+        [Required]
+        public Guid PublicationId { get; set; }
+        
+        [Required]
+        public PublicationRole Role { get; set; }
+        
+        public bool Accepted { get; set; }
+
+        public bool EmailSent { get; set; }
+
+        public DateTime Created { get; set; }
```

Comment:
Also added this to `UserReleaseInvite`.

I think it makes sense to keep UserReleaseInvite and UserPublicationInvite aligned in case we create an abstract class `UserResourceInvite` in the future, much like Dunc's `ResourceRole`.

Reply:
I've removed `Updated` from `UserPublicationInvite`, because it wasn't being used.